.circleci/config.yml

```
@@ -28,10 +29,14 @@ jobs:
             - run:
                 name: Run rustfmt
                 command: cargo fmt -- --check
-            - rust/clippy:
-                # Force a failure on clippy warnings
-                flags: "-- --deny warnings"
-                with_cache: false
+            # Run the commands without using the orbs since Clippy needs a little longer then 10 minutes
```

This note shocked me (that clippy needs so much time). After digging in, it looks like it's really the build time that is taking so long. The note would have been clearer to me if it said something like:
```suggestion
            # Run the sub-commands of the orb jobs separately, since the combined steps for Clippy can be long enough to trigger a timeout (>10min). Building the source is the main contributor.
```